packages/circuit-ui/components/Tooltip/Tooltip.tsx

```diff
@@ -14,24 +14,43 @@
  */
 
 import { css } from '@emotion/react';
-import styled from '@emotion/styled';
+import {
+  arrow,
+  autoUpdate,
+  offset,
+  Placement,
+  useFloating,
+} from '@floating-ui/react-dom';
+import { useRef } from 'react';
+
+import styled, { NoTheme, StyleProps } from '../../styles/styled';
+import { typography } from '../../styles/style-mixins';
+import { uniqueId } from '../../util/id';
 
-import type { StyleProps } from '../../styles/styled';
-import { typography, shadow } from '../../styles/style-mixins';
+export interface TooltipProps {
+  /**
+   * The text content of the tooltip.
+   */
+  text: string;
```

Comment:
Conventionally, we call props like this one `label` in Circuit UI.

```suggestion
  /**
   * The text content of the tooltip.
   */
  label: string;
```